### ISSUE TITLE
Annotated alias (PEP-593) is not supported

### ISSUE THREAD

**Issue by bpiwowar** [microsoft/pyright#1511]
**Describe the bug**
Annotated alias (with `TypeVar`) described in   [PEP 593](https://www.python.org/dev/peps/pep-0593/) is not supported

**To Reproduce**

```py3
from typing import TypeVar
from typing_extensions import Annotated

T = TypeVar("T")

paramHint = "hello"
Param = Annotated[T, paramHint]

x: Param[int]
```

**Expected behavior**

pyright should infer that the type of `x` is `int`, but it reports the error `TypeVar "Type[T@Param]" is not subscriptable and does not infer that x is an `int`.

**VS Code extension or command-line**
Using pyright as a vscode extension (pylance 2021.2.3)

**Comment by erictraut** [comment]
Thanks for reporting the issue. I overlooked that example in PEP 593 when I implemented the Annotated support in pyright. This will be addressed in the next release.

Here's [the change](https://github.com/microsoft/pyright/commit/b7090f81726ca56be2cd7193d171518280fa0f1e), if you're interested.

**Comment by bpiwowar** [comment]
Many thanks for fixing this issue so quickly!

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.113, which I just published. It will also be included in the next release of Pylance.
